test/test_torch.py

```diff
@@ -3776,12 +3775,31 @@ def test_trilu_indices(self):
         self.assertEqual(
             x.triu(0).nonzero().transpose(0, 1), torch.triu_indices(3, 3))
 
-    def test_triu(self):
-        x = torch.rand(SIZE, SIZE)
-        res1 = torch.triu(x)
-        res2 = torch.Tensor()
-        torch.triu(x, out=res2)
-        self.assertEqual(res1, res2, 0)
+    @staticmethod
+    def _test_triu_tril(self, cast):
```

Comment:
Is there another triu test somewhere? the original one is... really weird

Reply:
No, I removed the triu test. I seem to have missed the tril test while fixing a merge conflict